context/context.go

```diff
@@ -379,6 +380,18 @@ func (context *AptlyContext) GetPublishedStorage(name string) aptly.PublishedSto
 			if err != nil {
 				Fatal(err)
 			}
+		} else if strings.HasPrefix(name, "azure:") {
```

Comment:
This `azure:` seems to be related to [this string](https://github.com/aptly-dev/aptly/pull/413/files/fd66753b0b2c5cc937b4369f24ea6cc73a10f096#diff-0b71a2986f7c3b42cd53c374c70c4d0db8cade7dfddabaa530477c9600694f33R54) — doesn’t it need to be updated as well?

Reply:
No. They are different. The link is for string representation of Azure storage. Here we are checking user input, which needs to be lower case.